### PR TITLE
Create localize command

### PR DESCRIPTION
**Pull request by richardbporter**
This also runs after an sql-sync.

### CODE REVIEW

File: herky.drush.inc [hunk 1]
```diff
@@ -90,6 +90,28 @@ function drush_herky_sql_sync_pre_validate($source = NULL, $destination = NULL)
...
+    $self = drush_sitealias_get_record($destination);
+
+    // Drush assigns a group namespace and is smart enough to resolve @self.
+    if ($local['#group'] == $self['#group']) {
+      // If it matches, reassign $destination to the local alias name.
+      $destination = "{$group}.local";
+    }
+  }
+
+  _drush_herky_localize($destination);
```
Comment (joewhitsitt) [inline herky.drush.inc]: I think this needs to be `_herky_localize($destination);` otherwise i get a `call to undefined function _drush_herky_localize()`

**Comment by richardbporter** [comment]
Whoops. Didn't mean to assign you. This isn't complete yet.

**Comment by joewhitsitt** [comment]
you didn't i was anxious to peek!

**Comment by richardbporter** [comment]
@joewhitsitt I've got this set up to run automatically after an sql-sync command. I'm going to write tests tomorrow but if you wanted to try it out locally, you could. It won't work with remote sites b/c it is not deployed to the servers yet.

**Comment by richardbporter** [comment]
`drush hloc` or `drush @mysite.local hloc` or `drush sql-sync @mysite.prod @mysite.local`

**Comment by joewhitsitt** [comment]
i tested the prod to local sql-sync. seems to work just dandy!

**Comment by richardbporter** [comment]
If we decide post sql-sync should be automatic or opt-out, we'll probably want to add `$options['strict'] = FALSE;` to drushrc.php with Nile.

**Pull request merged**